platform/tron/api.go

```diff
@@ -47,11 +54,20 @@ func (p *Platform) GetTokenTxsByAddress(address, token string) (blockatlas.TxPag
 		return nil, err
 	}
 
+	if len(tokenTxs) == 0 {
+		return nil, err
+	}
+
 	var tokenInfo AssetInfo
 	info, err := p.client.GetTokenInfo(token)
 	if err != nil {
```

Comment:
can you add `len(info.Data) == 0` here too?

Reply:
https://github.com/trustwallet/blockatlas/pull/357/commits/f47a6982c4bc5af56d8b5fe366caf2e55eff9c15